.github/workflows/build.yml

```diff
@@ -53,8 +53,8 @@ jobs:
     strategy:
       matrix:
         include:
-          - platform: macos-latest
-            os: darwin
+          # - platform: macos-latest
```

Comment:
Is this intended to remove macos-latest?

Reply:
Temporary commit so I don't clog up runners. We should run the full suite at least one time with this change removed.

Separately I'll propose in Slack to move Windows and Mac integration tests to a scheduled job on the master branch.

Reply:
Excellent. Thanks.